### PR TITLE
GitHub Unity3D actions

### PR DESCRIPTION
**Pull request by ugnelis**
Solves #18.

Actions workflow based on https://github.com/webbertakken/unity-actions/blob/master/.github/workflows/main.yml
Regarding the Unity License, right now I've put my personal one in Github secrets.

I commented out Artifacts part. It might be needed in the near future.

### CODE REVIEW

**Comment by Edvinas01** [comment]
This closes #18

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+      # Test
+      - name: Run Tests
+        uses: webbertakken/unity-test-runner@v1.3
+        with:
+          projectPath: test-project
+          unityVersion: 2019.4.12f1
+
+      # Build
+      - name: Build Project
```
Comment (Edvinas01) [inline .github/workflows/main.yml]: Is this step necessary?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      # Cache
+      - uses: actions/cache@v1.1.0
+        with:
+          path: test-project/Library
+          key: Library-test-project-WebGL
```
Comment (Edvinas01) [inline .github/workflows/main.yml]: I think the `WebGL` suffix doesn't make much sense here.

**Review by Edvinas01 — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (ugnelis) [reply .github/workflows/main.yml]: Good point.

File: .github/workflows/main.yml (hunk 2 above)
Comment (ugnelis) [reply .github/workflows/main.yml]: Ahh, yes, this is not necessary right now because if I am right, we don't have any libraries right now

**Pull request merged**